pkgs/desktops/gnome-3/3.24/apps/seahorse/default.nix

```diff
@@ -16,11 +16,12 @@ stdenv.mkDerivation rec {
   buildInputs = [ pkgconfig gtk3 glib intltool itstool gnome3.gcr
                   gnome3.gsettings_desktop_schemas wrapGAppsHook gnupg
                   gdk_pixbuf gnome3.defaultIconTheme librsvg gpgme
-                  libsecret avahi libsoup p11_kit vala_0_32
+                  libsecret avahi libsoup p11_kit vala_0_32 gnome3.dconf
                   openssh ];
 
   preFixup = ''
     gappsWrapperArgs+=(
+      --prefix GIO_EXTRA_MODULES : "${gnome3.dconf}/lib/gio/modules"
```

Comment:
The memory commit should not be needed, it is already included in staging. I removed this superfluous change there.